pootle/apps/pootle_fs/management/commands/fs_commands/rm.py

```diff
@@ -19,4 +19,4 @@ def add_arguments(self, parser):
             "--force",
             action="store_true",
             dest="force",
-            help=("Stage files and stores that are conflicting for removal"))
+            help=("Stage files and/or stores for removal"))
```

Comment:
ok. I thought that it was possible to remove any file/store independently of conflict/untracked status.